codecov.yml

```diff
@@ -1,6 +1,6 @@
 ignore:
   - "Directions Example"
-  - "Tests"
+  - "MapboxDirectionsTests"
```

Comment:
#360 originally intended to omit MapboxDirectionsTests from code coverage metrics, but when #362 renamed this folder to Tests, test code inflated test coverage by around 11%. We’ll need to adjust reapply this change in a separate commit outside of this PR to avoid getting dinged by the (justifiable) omission of test code from code coverage metrics.